### DISCUSSION TITLE
SavvyCAN & RP1210 adapter

### DISCUSSION THREAD

**Discussion by Fang717** [collin80/SavvyCAN#601] [General]
Hello,
I wonder if it possible to use SavvyCAN with RP1210 adapters? /such as Nexiq USB-Link 2/
Thank you

**Comment by collin80** [comment]
There is some support for J2534 which that device says it supports. However, the default speed is 250k and I'm hearing reports that this tends to break things and that the default should be 500k for most uses. I can change the default (in fact, I'll do that right now) but still support for J2534 is somewhat experimental.
